bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueBindingConstants.java

```diff
@@ -107,4 +110,29 @@ public class HueBindingConstants {
 
     // Config status messages
     public static final String IP_ADDRESS_MISSING = "missing-ip-address-configuration";
+
+    // thing types for CLIP 2
+    public static final ThingTypeUID THING_TYPE_CLIP2 = new ThingTypeUID(BINDING_ID, "clip2");
+    public static final ThingTypeUID THING_TYPE_DEVICE = new ThingTypeUID(BINDING_ID, "device");
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES2 = Set.of(THING_TYPE_CLIP2, THING_TYPE_DEVICE);
+
+    // channels for CLIP 2
+    public static final String CHANNEL_BUTTON_LAST_EVENT = "button_last_event";
+    public static final String CHANNEL_TEMPERATURE_ENABLED = "temperature_enabled";
+    public static final String CHANNEL_MOTION = "motion";
+    public static final String CHANNEL_MOTION_ENABLED = "motion_enabled";
+    public static final String CHANNEL_LIGHT_LEVEL_ENABLED = "light_level_enabled";
+    public static final String CHANNEL_ZIGBEE_STATUS = "zigbee_status";
+    public static final String CHANNEL_BUTTON_TRIGGER_EVENT = "button_trigger_event";
+    public static final String CHANNEL_ROTARY_STEPS = "rotary_steps";
+
+    // property names for CLIP 2
+    public static final String PROPERTY_RESOURCE_ID = "resourceId";
+    public static final String PROPERTY_RESOURCE_TYPE = "resourceType";
+    public static final String PROPERTY_RESOURCE_NAME = "resourceName";
+    public static final String PROPERTY_RESOURCE_ARCHETYPE = "resourceArchetype";
+    public static final String PROPERTY_PRODUCT_NAME = "productName";
+    public static final String PROPERTY_PRODUCT_ARCHETYPE = "productArchetype";
+    public static final String PROPERTY_PRODUCT_CERTIFIED = "productCertified";
```

Comment:
Umm. I was trying to follow the rule "do not mess with the old binding". For this reason, I have tried wherever possible to only add classes, and not unnecessarily change any existing classes. (This would make the review much easier for you guys, since the number of changed files is fewer..) Also the comment line above the V1 properties says "Thing configuration properties" is actually wrong because these are actually (static) properties and not related to configuration at all.

Reply:
That's a noble intention more than a strict rule. 🙂 In this case eliminating duplication seems preferable. I see only duplication of product name, so would propose to go with your new name and refactor the two existing usages `PRODUCT_NAME` to also use `PROPERTY_PRODUCT_NAME`.

Reply:
I will do what you request.

.. but it is worth noting that there is really no duplication, since Java string code optimisation will ensure that both of the so called 'duplicated' constants will compile to the same pointer reference to one identical string.

Reply:
My comment was towards code duplication, not resulting byte-code - i.e. consistency and maintainability.